README.md

```diff
@@ -102,6 +102,33 @@ canvas.course.migrate(1, 110, (error, results) => {
 });
 ```
 
+
+#### `course.discuss(course, params, callback)`
+
+> Create an assignment in the desired course.
```

Comment:
This looks like a left over from a copy and paste - can you please update it to match the nature of the method?